### DISCUSSION TITLE
[1260번] DFS와 BFS (feat: 동적할당 오버헤드 및 고정크기배열)

### DISCUSSION THREAD

**Discussion by stopmin** [BDD-CLUB/challenge-every-week-posting#421] [3. CS(컴퓨터 공학) 관련 글 모음]
하핫 글 한개를 더 쓰게 됐네요!!! 

종만북만 읽긴 심심해서 조료구조 종강하고 한번도 알고리즘 그래프 문제를 풀어본 적이 없어서 오랜만에 bfs/dfs문제를 풀어봤숩니다!! 제일 쉬운 문제긴 한데,, 구현하는데는 별로 안걸렸는데 알고보니 9개월 전에 제가 풀었더라고요!!! 

근데 메모리랑 실행시간이 너무너무 차이가 많이 나서..충격받은 마음에 왜 그런지에 대해서 글을 썼습니다. (9개월에 나는 어떻게 저렇게 짤 생각을 한거쥐..) 

너무 간단한 문제지만 이렇게 비교해보니... 재미있긴 한데 보면 볼수록 마음이 아프네요. 이런 작은 문제에서도 9개월 전의 나와 이렇게나 비교되다니...
종만북 열심히 읽으면서 공부해서 9개월을 뛰어넘겠습니다!!!

- [[1260번] DFS와 BFS (feat: 동적할당 오버헤드 및 고정크기배열)
](https://stopmin.tistory.com/entry/1260%EB%B2%88-DFS%EC%99%80-BFS-feat-%EB%8F%99%EC%A0%81%ED%95%A0%EB%8B%B9-%EC%98%A4%EB%B2%84%ED%97%A4%EB%93%9C-%EB%B0%8F-%EA%B3%A0%EC%A0%95%ED%81%AC%EA%B8%B0%EB%B0%B0%EC%97%B4-%EC%BA%90%EC%8B%9C)

**Comment by pnudpwls1103** [comment]
개선을 위해 분석한 점이 멋쪄요!!👍
